File: questModule/src/main/java/ru/nekit/android/qls/quest/mediator/shared/content/EmptyQuestContentMediator.java
package ru.nekit.android.qls.quest.mediator.shared.content;

import android.view.View;
import android.widget.EditText;

public class EmptyQuestContentMediator extends AbstractQuestContentMediator {

    @Override
    public EditText getAnswerInput() {
        return null;
    }

    @Override
    public boolean includeInLayout() {
        return true;
    }

    @Override
    public View getView() {
        return null;
    }

    @Override
    public void updateSize() {

    }
}

File: questPresentation/src/main/java/ru/nekit/android/qls/quest/view/mediator/content/EmptyQuestContentMediator.kt
package ru.nekit.android.qls.quest.view.mediator.content

import android.view.View
import android.widget.EditText

//ver 1.0
class EmptyQuestContentMediator : SimpleContentMediator() {

    override val view: View? = null

    override val answerInput: EditText? = null

    override fun includeInLayout(): Boolean {
        return true
    }

    override fun updateSize() {

    }
}